editor/plugins/script_text_editor.h

```diff
@@ -143,15 +143,18 @@ class ScriptTextEditor : public ScriptEditorBase {
 	};
 
 protected:
-	static void _code_complete_scripts(void *p_ud, const String &p_code, List<String> *r_options, bool &r_force);
+
```

Comment:
`clang-format` complains about this empty line: https://travis-ci.org/godotengine/godot/jobs/554037059

Reply:
Fixed.
Is it possiable to remind me to format the code when I commit with git ?

Reply:
Yes, you can install the pre-commit hooks in https://github.com/godotengine/godot/tree/master/misc/hooks
You might need to edit https://github.com/godotengine/godot/blob/master/misc/hooks/pre-commit-clang-format#L20 if `which clang-format` is not sufficient to find a recent version of `clang-format` on your system (8.0 or later).

Reply:
If you have the Python `pygments` package, you can also use colorized diff output by uncommenting https://github.com/godotengine/godot/blob/master/misc/hooks/pre-commit-clang-format#L38